README.md

```diff
@@ -10,6 +10,14 @@ These release notes will contain a sum-up of all customer-relevant changes that
 
 If you're viewing this on GitHub, please navigate to https://docs.fiskaltrust.cloud for a more convenient view of this content and open the section of interest to the left. If you have any questions regarding the changes described here, please directly reach out to us via [info@fiskaltrust.de](mailto:info@fiskaltrust.de) / [info@fiskaltrust.at](mailto:info@fiskaltrust.at) / [info@fiskaltrust.fr](mailto:info@fiskaltrust.fr).
 
+## September 15, 2021
+
+### Middleware 1.3.23 (Germany)
+In this version of the Middleware, we added the possibility to easily identify mandatory and optional signature items when printing receipts. Additionally, the Middleware is now able to automatically update the _Fiskal Cloud Connector_, and several bugs and stability issues have been resolved.
+
+[Middleware 1.3.23 Release Notes](middleware/middleware-1.3.23.md)
+
+
```

Comment:
do we always have two linebreaks?

Reply:
Yep, only before new days though - it's easier to read through the file IMO 😄